pkg/tenant/db/db.go

```diff
@@ -302,8 +303,11 @@ func deleteRequest(ctx context.Context, namespace string, key []byte) (err error
 	return nil
 }
 
-// List retrieves a pagination cursor.
-func List(ctx context.Context, prefix, seekKey []byte, namespace string, onListItem OnListItem, c *pg.Cursor) (cursor *pg.Cursor, err error) {
+// List iterates over items in the database and calls the onListItem function for each
+// of them. If onListItem returns ErrListBreak then the iteration will stop. If there
+// are more items than the page size in the cursor then a new cursor is returned for
+// the next page of items.
+func List(ctx context.Context, prefix []byte, namespace string, onListItem OnListItem, c *pg.Cursor) (cursor *pg.Cursor, err error) {
```

Comment:
I don't remember why we were passing in the seek key rather than just using the cursor so I removed it from the interface in favor of the cursor.